recipes/openmpi/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from os import path
+
+
+class OpenmpiConan(ConanFile):
+    name = "openmpi"
+    license = "3-clause BSD license"
```

Comment:
```suggestion
BSD-3-Clause
```
License must be a short identifier in spdx database.
https://spdx.org/licenses/BSD-3-Clause.html

Reply:
Oh! Thank you. I was not sure about this :) Thanks